recipe/build.sh

```diff
@@ -12,9 +12,11 @@ if [[ $(uname -s) == Darwin ]]; then
   # this works for macOS 10.12
   # export cc_opt="$cc_opt -O2 -g -pipe -Wall -Wp,-D_FORTIFY_SOURCE=2 -fexceptions -fstack-protector-strong --param=ssp-buffer-size=4 -fno-strict-overflow -m64 -mtune=generic -fPIC"
 
-  export cc_opt="$cc_opt $CFLAGS"
+  export cc_opt="$cc_opt -O2 -g -pipe -Wall -Wp,-D_FORTIFY_SOURCE=2 -fexceptions -fstack-protector --param=spp-buffer-size=4 -fno-strict-overflow -m64 -mtune=generic -fPIC"
```

Comment:
Why do we need to specify these explicitly?

Reply:
I reverted my changes and now there is:
`export cc_opt="$cc_opt $CFLAGS"`